package.json

```diff
@@ -142,7 +142,7 @@
     "newrelic-naming-rules": "./bin/test-naming-rules.js"
   },
   "dependencies": {
-    "@grpc/grpc-js": "1.0.5",
+    "@grpc/grpc-js": "^1.1.7",
```

Comment:
hmm, I thought so based on https://github.com/grpc/grpc-node/releases/tag/grpc%401.24.3. But it doesn't seem like grpc-js uses that as a dependency, so it could possibly be dropped back to 1.0.5.

Reply:
Same error with the version reverted back to 1.0.5, unfortunately